runner/cf-driver/prepare.sh

```diff
@@ -117,10 +117,12 @@ install_dependencies () {
     # Of course, RedHat/UBI will need more help to add RPM repos with the correct
     # version. TODO - RedHat support
     echo "[cf-driver] Ensuring git, git-lfs, and curl are installed"
-    cf ssh "$container_id" -c '(which git && which git-lfs && which curl) || \
+    cf ssh "$container_id" --request-pseudo-tty \
```

Comment:
question: what's the pseudo-tty's purpose here?

I do remember you had an issue when you were trying locally and this seemed to have an impact, but w/ the runner it doesn't make a difference for me as far as I can tell.

Reply:
I think the `request` bit is actually being denied. I don't know that for sure, but when I try `force...` instead it just hangs.